### PR TITLE
Clean up for DG

### LINKED ISSUE #587: DG: Implementation(Request), Documentation-logging, Appendix , A, B

### LINKED ISSUE #598: Update version number
image

### PR DESCRIPTION
**Pull request by tlylt**
- Minor clean up for Documentation, logging, testing, configuration, dev-ops
- Fixes #598 
- Fixes #587

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -411,21 +411,25 @@ at the destroy marker (X) but due to a limitation of PlantUML, the lifeline reac
...
 - requires testing of APIs in projects or work-related tasks
+- needs to learn and interact with APIs
+- prefers a portable and lightweight application
 
 **Value proposition**:
 
-- beautify response
-- save requests for quick execution again
-- general analysis/recommendation system based on certain metrics
+- beautify and highlight key matrix of API responses
```
Comment (tjtanjin) [inline docs/DeveloperGuide.md:424]: Do you mean metrics? :P

File: docs/index.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ title: imPoster
 </p>
 
 <h1 class="post-title">{{ page.title | escape }}</h1>
-<h3 class="post-subtitle">v1.3</h3>
+<h3 class="post-subtitle">v1.4</h3>
```
Comment (tjtanjin) [inline docs/index.md:11]: I think we can put v1.4b since this is the final milestone :D

**Review by tjtanjin — APPROVED** [review]
LGTM, just some nits to fix :)

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (tlylt) [reply docs/DeveloperGuide.md:424]: Yes sorry, will fix

File: docs/index.md (hunk 2 above)
Comment (tlylt) [reply docs/index.md:11]: ok sure

**Pull request merged**